src/main/java/org/gitlab4j/api/models/User.java

```diff
@@ -15,98 +14,27 @@ public String getExternUid() {
         return this.externUid;
     }
 
-    public User withEmail(String email) {
-        setEmail(email);
-        return this;
-    }
-
-    public User withName(String name) {
-        setName(name);
-        return this;
-    }
-
-    public User withUsername(String username) {
-        setUsername(username);
-        return this;
-    }
-
-    public User withSkype(String skype) {
-        setSkype(skype);
-        return this;
-    }
-
-    public User withLinkedin(String linkedIn) {
-        setLinkedin(linkedIn);
-        return this;
-    }
-
-    public User withTwitter(String twitter) {
-        setTwitter(twitter);
-        return this;
-    }
-
-    public User withWebsiteUrl(String websiteUrl) {
-        setWebsiteUrl(websiteUrl);
-        return this;
-    }
-
-    public User withOrganization(String organization) {
-        setOrganization(organization);
-        return this;
-    }
-
+    /**
+     * @deprecated Replaced by {@link #withProjectsLimit(Integer)}
+     * @see #withProjectsLimit(Integer) 
+     */
+    @Deprecated
     public User withProjectLimit(Integer projectsLimit) {
```

Comment:
Is there a reason to not remove this like the other with* methods that were removed?

Reply:
@gmessner I didn't want to break the API. I don't know, if someone uses that method. But I can also remove these methods instead of deprecating them, if you like.

Reply:
@mdeknowis 
I got it, I missed the fact that an "s" was missing. Leave it in as deprecated.